### PR TITLE
add chatbot api

### PR DESCRIPTION
**Pull request by archaeopteryx1**
This PR adds a new `chatbot` category and a powerful chatbot api to the api list.

### CODE REVIEW

File: src/content/apis/chatbot/brainshop-ai.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: BrainShop.AI
+description: A highly customizable, multilingual chatbot API by Acobot
+categories:
+  - chatbot
```
Comment (TheLearneer) [inline src/content/apis/chatbot/brainshop-ai.md:5]: I am not sure if we will many chatbot APIs, so for the time being, can we just change it to other categories.
```suggestion
  - entertainment
  - miscs
```

File: docker-compose.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: '3'
+version: "3"
```
Comment (TheLearneer) [inline docker-compose.yml:1]: Please remove this change

File: docker-compose.yml [hunk 3]
```diff
@@ -11,4 +11,4 @@ services:
       - .:/app
       - /app/node_modules
     ports:
-      - "3000:3000"
\ No newline at end of file
+      - "3000:3000"
```
Comment (TheLearneer) [inline docker-compose.yml:14]: Please remove this change

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Just as mentioned, please do update the categories

File: docker-compose.yml (hunk 3 above)
Comment (archaeopteryx1) [reply docker-compose.yml]: prettier 💯

Comment (TheLearneer) [reply docker-compose.yml]: ohh 😝

**Review by TheLearneer — APPROVED** [review]
LGTM, thank you for your contribution @archaeopteryx1

**Pull request merged**